### ISSUE TITLE
Add Missing Methods for Messages

### ISSUE THREAD

**Issue by Cliftonz** [novuhq/novu-python#78]
Some endpoints of the following section can't be hit from this SDK. 

We would like to ensure that all endpoints can be hit. For this issue, we would like to build out the Messages

Some of these methods may already exist.  If so no need to change/update them, just point out that they already exist and we will close the issue.  This is just to bring every SDK we have on par with all the methods available.

Check all the endpoints [here](https://api.novu.co/api#/Workflow%20groups/NotificationGroupsController_updateNotificationGroup) and add the missing ones.

**Comment by mr-palindrome** [comment]
Hey! I am looking to contribute on novu python sdk for the occasion of hacktoberfest. Can you please elaborate what do I need to do under this issue. Do I need to create methods for those endpoints which are missing or just need to add them somewhere?

**Comment by Cliftonz** [comment]
@mr-palindrome  Yes you would need to create and add the two methods for these two endpoints.
https://api.novu.co/api#/Messages

**Comment by mr-palindrome** [comment]
> @mr-palindrome Yes you would need to create and add the two methods for these two endpoints. https://api.novu.co/api#/Messages

Hey @Cliftonz I checked and verified that both message requests are already available and operational! Let me look over the other issues, and if there are any that are missing, I'll start contributing to them.
